Exiled.Loader/AutoUpdateFiles.tt

```diff
@@ -0,0 +1,24 @@
+<#@ template language="C#" #>
+<#@ assembly name="$(EXILED_REFERENCES)\Assembly-CSharp-Publicized.dll" #>
+// -----------------------------------------------------------------------
+// <copyright file="AutoUpdateFiles.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Loader
+{
+    using System;
+
+    /// <summary>
+    /// Class automatically update with Reference use for generate exiled.
+    /// </summary>
+    public static class AutoUpdateFiles
+    {
+        /// <summary>
+        /// Gets SCPSL Version use to generate exiled.
```

Comment:
```suggestion
        /// Gets what SCP: SL version generated Exiled.
```

Reply:
```suggestion
        /// Gets which SCP: SL version generated Exiled
```

Reply:
I wanted to use "which" too, but I wasn't too sure about it.